sufia-models/app/models/concerns/sufia/collection_behavior.rb

```diff
@@ -15,10 +15,35 @@ def update_permissions
       self.visibility = "open"
     end
 
-    # Compute the sum of each file in the collection
-    # Return an integer of the result
+    # Compute the sum of each file in the collection using Solr to
+    # avoid having to hit Fedora 
     def bytes
-      members.reduce(0) { |sum, gf| sum + gf.content.size.to_i }
+      query = ActiveFedora::SolrQueryBuilder.construct_query_for_rel(has_model: file_model)
+      args = {
+        fq: "{!join from=hasCollectionMember_ssim to=id}id:#{id}",
+        fl: "id, #{file_size_field}",
+        rows: members.count
+      }
+
+      files = ActiveFedora::SolrService.query(query, args)
+      files.reduce(0) { |sum, f| sum += f[file_size_field].to_i }
+    end
+
+    protected
+    # Field to look up when locating the size of each file in Solr.
```

Comment:
Indent all the protected methods two more spaces. (This is a Rails convention, though not a Ruby convention, IIRC.)